changelog/434.improvement.md

```diff
@@ -0,0 +1,2 @@
+- The action server can now listen on a specific network address using the environment variable `SANIC_HOST` instead
+- Corrected url which is logged at action server startup
```

Comment:
Is there meant to be a fullstop between `instead` and `Corrected?

Reply:
Thanks for reviewing @wochinge. The answer is no and I corrected the text